pkg/provision/providers/vm/network.go

```diff
@@ -129,6 +131,65 @@ func (p *Provisioner) CreateNetwork(ctx context.Context, state *State, network p
 		return fmt.Errorf("error parsing VM CNI config: %w", err)
 	}
 
+	// configure bridge interface with network chaos if flag is set
+	if network.NetworkChaos {
+		link, err := netlink.LinkByName(state.BridgeName)
+		if err != nil {
+			return fmt.Errorf("could not get link: %v", err)
+		}
+
+		fmt.Fprintln(options.LogWriter, "network chaos enabled on interface:", state.BridgeName)
+
+		if network.Bandwidth != 0 {
```

Comment:
I guess it makes sense to return an error if both bandwidth & any of jitter/latency is set.

Reply:
I added a line that prints a log message saying "bandwidth is set. No other chaos options will be used." 

Another option could be to set two flags instead of one to enable network chaos. One for bandwidth (tbf) qdiscs and one for the other other (netem) qdiscs. Currently, you set the flag `--network-chaos-enabled` and then set your other flags such as `network-jitter`, `network-latency`, `network-packet-loss`, `network-packet-reorder`, `network-packet-corrupt`, `network-bandwidth` and if bandwidth is set, it will just not apply any of the other flags.